riocli/jsonschema/schemas/package-schema.yaml

```diff
@@ -137,6 +137,65 @@ definitions:
           memory:
             type: number
             minimum: 0
+      livenessProbe:
+        type: object
+        properties:
+          http:
+            type: object
+            properties:
+              path:
+                type: string
+                description: Path to access for the HTTP probe.
+              port:
+                type: integer
+                description: Port number for the HTTP probe.
+            required:
+              - path
+              - port
+          exec:
+            type: object
+            properties:
+              command:
+                type: array
+                items:
+                  type: string
+                description: Command to execute for the Exec probe.
+            required:
+              - command
+          tcp:
+            type: object
+            properties:
+              port:
+                type: integer
+                description: Port number for the TCP probe.
+            required:
+              - port
+          initialDelaySeconds:
+            type: integer
+            description: Number of seconds after the container has started before liveness probes are initiated.
+          timeoutSeconds:
+            type: integer
+            default: 1
+            description: Number of seconds after which the probe times out.
+          periodSeconds:
+            type: integer
+            default: 1
+            description: How often (in seconds) to perform the probe.
+          successThreshold:
+            type: integer
+            default: 3
+            description: Minimum consecutive successes for the probe to be considered successful after having failed.
+          failureThreshold:
+            type: integer
+            default: 3
+            description: Minimum consecutive failures for the probe to be considered failed after having succeeded.
+        oneOf:
+          - required:
+              - http
+          - required:
+              - tcp
+          - required:
+              - exec
```

Comment:
Create a new definition for `livenessProbe`.  Use `ref` to use it under device executables.